app/src/main/java/ch/epfl/sweng/SDP/MainActivity.java

```diff
@@ -9,6 +9,10 @@
 import ch.epfl.sweng.SDP.firebase.Database;
 import ch.epfl.sweng.SDP.home.HomeActivity;
 
+import ch.epfl.sweng.SDP.shop.ShopActivity;
+
+import com.google.firebase.FirebaseApp;
+
```

Comment:
Are those import really needed ? (com.google.firebase.FirebaseApp is already imported)